src/docs/references/transaction-process-actions/index.md

```diff
@@ -915,12 +915,32 @@ requires that the transition is made from a trusted context.
 
 #### :action/stripe-create-payment-intent
 
-Action for creating a Stripe Payment Intent for the transaction.
+Action for creating a Stripe PaymentIntent for the transaction. This
+action supports card payments or equivalent (such as using wallets like
+Google Pay and Apple Pay).
 
 Payment Intents are the main supported way to collect payments.
 Transaction processes need to use them especially if they want to be
 SCA-compatible.
 
+After the PaymentIntent is created, it's ID and client secret are
+temporarily accessible in the Flex transaction protected data under the
+`stripePaymentIntents` key in the following form:
+
+```json
+{
+  "default": {
+    "stripePaymentIntentId": "pi_1EXSEzLSea1GQQ9x5PnNTeuS",
+    "stripePaymentIntentClientSecret": "pi_1EXSEzLSea1GQQ9x5PnNTeuS_secret_xxxxxxxxxxxxxxxxxxxxxxxxx"
```

Comment:
Btw. does it matter if these are obfuscated or not?
Here's one shown as it is:
https://deploy-preview-399--sharetribe-flex-docs-site.netlify.app/docs/background/payment-intents/#step-2-collect-payment-information-and-handle-customer-actions

Reply:
It doesn't really matter. Do you think obfuscating that has any downside? The format of that value anyway is not controlled by us and can change, so people shouldn't rely on what exactly it looks like.